### PR TITLE
Adding limits concerning RWX & total of 25 attached

### PR DESCRIPTION
**Pull request by mhurtrel**

### CODE REVIEW

File: pages/platform/kubernetes-k8s/known-limits/guide.en-gb.md [hunk 1]
```diff
@@ -90,13 +90,16 @@ Please refrain from adding private networks to your working nodes instances.
...
 
 ```bash
 The PersistentVolumeClaim "mysql-pv-claim" is invalid: spec.resources.requests.storage: Forbidden: field can not be less than previous value
 ```
-
 For more details, please refer to the [Resizing Persistent Volumes documentation](../resizing-persistent-volumes/) documentation.
+
+The Persistent Volumes are using our Cinder-based block-storage solution though the Cinder CSI.
+A worker node can get attached to a maximum of 25 persistent volumes, and a persistent volumes can only be attached to a single worker node.
+You can manually [configure multi-attach persistent volumes with NAS-HA ](../Configuring-multi-attach-persistent-volumes-with-ovhcloud-nas-ha/) .
```
Comment (XavierDuthil) [inline pages/platform/kubernetes-k8s/known-limits/guide.en-gb.md]:
```suggestion
You can manually [configure multi-attach persistent volumes with NAS-HA](../Configuring-multi-attach-persistent-volumes-with-ovhcloud-nas-ha/).
```

File: pages/platform/kubernetes-k8s/known-limits/guide.en-gb.md [hunk 2]
```diff
@@ -90,13 +90,16 @@ Please refrain from adding private networks to your working nodes instances.
...
 Kubernetes `Persistent Volume Claims` resizing only allows to __expand__ volumes, not to __decrease__ them.  
 If you try to decrease the storage size, you will get a message like:
 
 ```bash
 The PersistentVolumeClaim "mysql-pv-claim" is invalid: spec.resources.requests.storage: Forbidden: field can not be less than previous value
 ```
-
 For more details, please refer to the [Resizing Persistent Volumes documentation](../resizing-persistent-volumes/) documentation.
+
+The Persistent Volumes are using our Cinder-based block-storage solution though the Cinder CSI.
```
Comment (XavierDuthil) [inline pages/platform/kubernetes-k8s/known-limits/guide.en-gb.md]:
```suggestion
The Persistent Volumes are using our Cinder-based block-storage solution through Cinder CSI.
```

File: pages/platform/kubernetes-k8s/known-limits/guide.en-gb.md [hunk 3]
```diff
@@ -90,13 +90,16 @@ Please refrain from adding private networks to your working nodes instances.
...
+## Persistent Volumes
 
 Kubernetes `Persistent Volume Claims` resizing only allows to __expand__ volumes, not to __decrease__ them.  
 If you try to decrease the storage size, you will get a message like:
 
 ```bash
 The PersistentVolumeClaim "mysql-pv-claim" is invalid: spec.resources.requests.storage: Forbidden: field can not be less than previous value
 ```
-
 For more details, please refer to the [Resizing Persistent Volumes documentation](../resizing-persistent-volumes/) documentation.
```
Comment (XavierDuthil) [inline pages/platform/kubernetes-k8s/known-limits/guide.en-gb.md]:
```suggestion
For more details, please refer to the [Resizing Persistent Volumes documentation](../resizing-persistent-volumes/).
```

**Review by XavierDuthil — CHANGES_REQUESTED** [review]

File: pages/platform/kubernetes-k8s/known-limits/guide.en-gb.md [hunk 4]
```diff
@@ -90,13 +90,16 @@ Please refrain from adding private networks to your working nodes instances.
...
+
+The Persistent Volumes are using our Cinder-based block-storage solution though the Cinder CSI.
+A worker node can get attached to a maximum of 25 persistent volumes, and a persistent volumes can only be attached to a single worker node.
```
Comment (jMonsinjon) [inline pages/platform/kubernetes-k8s/known-limits/guide.en-gb.md]:
```suggestion
A worker node can get attached to a maximum of 25 persistent volumes, and a persistent volume can only be attached to a single worker node.
```

**Comment by OVH-Team-Guides** [comment]
Thanks for your contribution @mhurtrel, @jMonsinjon and @XavierDuthil. Merge is done on develop. Merge on master will happen tomorrow morning.

**Pull request merged**